red/runtime/nodes/flows/index.js

```
@@ -108,7 +139,115 @@ function load(forceStart) {
+                if (activeFlowConfig) {
+                    if (activeFlowConfig.allNodes) {
+                        for (var nodeId in activeFlowConfig.allNodes) {
+                            delete activeFlowConfig.allNodes[nodeId];
+                        }
+                    }
+                    if (activeFlowConfig.subflows) {
+                        for (var subflowId in activeFlowConfig.subflows) {
+                            if (activeFlowConfig.subflows[subflowId]) {
+                                var subflow = activeFlowConfig.subflows[subflowId];
+                                if (subflow.nodes) {
+                                    for (var nodeId in subflow.nodes) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'nodeId'. Consider to delete it or rename it.

'nodeId' is defined here:


(lint/suspicious/noRedeclare)